src/intrinsic/x86.rs

```diff
@@ -0,0 +1,766 @@
+"llvm.x86.avx512.mask.pcmpeq.q.256" => "__builtin_ia32_pcmpeqq256_mask"
```

Comment:
Could this list be sorted?

Reply:
I was also thinking the same. I'm currently updating the script generating the instructions.